### PR TITLE
Build gppc(Greenplum Partner Connector) along with GPDB

### PR DESCRIPTION
**Pull request by kaknikhil**
We used to build gppc outside of gpdb (against an old version) but there is no reason to not open source it and build it along with the gpdb source code. This is an attempt to merge the gppc code and move all the gppc tinc tests to ICG . This is still a WIP and we haven't moved over any of the TINC tests yet , just wanted to get some feedback on the changes so far.  

Authors:
Nikhil Kak & Yandong Yao

### CODE REVIEW

**Comment by ivannovick** [comment]
LGTM!  Important to do this PR and get the code merged in with rest of GPDB and ICG so it can be maintained

File: src/interfaces/gppc/test/gppc_test.c [hunk 1]
```diff
@@ -0,0 +1,962 @@
+/*-------------------------------------------------------------------------
+ *
+ * gppc_client.c
```
Comment (danielgustafsson) [inline src/interfaces/gppc/test/gppc_test.c]: Nitpicking, but this comment is clearly misleading.

**Comment by yydzero** [comment]
pushed with @danielgustafsson 's suggestion

**Comment by xinzweb** [comment]
Did this passed the build? So far, this break the build. This both reproed on my box, in the build lab, and also on @armenatzoglou box.

**Comment by yydzero** [comment]
@xinzweb could you please share your failure?  it passed our local machine. I will monitor build also.

**Comment by yydzero** [comment]
I see the error, thanks, will fix it soon. Sorry for this, it passed on our
RHEL7 dev box and we committed.

On Tue, Jan 26, 2016 at 12:46 PM, Xin Zhang <notifications@github.com>
wrote:

> Did this passed the build? So far, this break the build. This both reproed
> on my box, in the build lab, and also on @armenatzoglou
> <https://github.com/armenatzoglou> box.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/greenplum-db/gpdb/pull/290#issuecomment-175221926>.
>



-- 
Best Regards,
Yandong

**Comment by yydzero** [comment]
Fixed

**Comment by xinzweb** [comment]
Thanks a lot.

**Pull request closed without merging**